source/agora/consensus/data/Block.d

```diff
@@ -123,17 +102,16 @@ unittest
     PublicKey pubkey = PublicKey.fromString(address);
 
     // above parts not @safe/@nogc yet
-    () @safe @nogc nothrow
+    () @safe
```

Comment:
The test was specifically designed to check for `@nogc/nothrow`-ness, so please revert this and the change to `assert`. Hopefully we'll get `-checkaction=context` soon-ish and will no more needs hacks like this.